src/main/webapp/app/modules/administration/user-management/user-management-update.tsx

```diff
@@ -47,137 +72,234 @@ export const UserManagementUpdate = () => {
+                      value: true,
+                      message: translate('global.messages.validate.email.required'),
+                    },
+                    minLength: {
+                      value: 5,
+                      message: translate('global.messages.validate.email.minlength'),
+                    },
+                    maxLength: {
+                      value: 254,
+                      message: translate('global.messages.validate.email.maxlength'),
+                    },
+                    validate: v => isEmail(v) || translate('global.messages.validate.email.invalid'),
+                  }}
+                  onChange={e => {
+                    setUserEmail(e.target.value);
+                  }}
+                />
+              </Col>
+              <Col md="6">
+                <ValidatedField
+                  type="select"
+                  name="langKey"
+                  label={translate('userManagement.langKey')}
+                  onChange={e => {
+                    setUserLangKey(e.target.value);
+                  }}
+                >
+                  {locales.map(locale => (
+                    <option value={locale} key={locale}>
+                      {languages[locale].name}
+                    </option>
+                  ))}
+                </ValidatedField>
+              </Col>
+              <Col md="6">
+                <div className="d-flex flex-column gap-2">
+                  <label>{translate('userManagement.profiles')}</label>
+                  <MultiSelect
+                    name="authorities"
+                    options={authorities}
+                    value={userAuthorities}
+                    onChange={e => {
+                      setUserAuthorities(e.value);
+                    }}
+                  />
+                </div>
+                {/* <ValidatedField type="select" name="authorities" multiple label={translate('userManagement.profiles')}>
+                  {authorities.map(role => (
+                    <option value={role} key={role}>
+                      {role}
+                    </option>
+                  ))}
+                </ValidatedField> */}
+              </Col>
+              {hasAnyAuthority(userAuthorities, [AUTHORITIES.STUDENT]) ? (
+                <Col md="6">
+                  <ValidatedField
+                    type="text"
+                    name="studentId"
+                    label={translate('userManagement.studentId')}
+                    validate={{
+                      maxLength: {
+                        value: 10,
+                        message: translate('entity.validation.maxlength', { max: 10 }),
+                      },
+                    }}
+                    value={userStudentId}
+                    onChange={e => {
+                      setUserStudentId(e.target.value);
+                    }}
+                  />
+                </Col>
               ) : null}
```

Comment:
use EMPTY_STRING instead of null

Reply:
I do not want to render any element so I put null value here